ctapipe/containers.py

```diff
@@ -1044,3 +1088,86 @@ class ArrayEventContainer(Container):
         "Container for calibration coefficients for the current event",
     )
     mon = Field(MonitoringContainer(), "container for event-wise monitoring data (MON)")
+
+
+class SchedulingBlockContainer(Container):
+    """Stores information about the scheduling block. This is a simplified version
+    of the SB model, only storing what is necessary for analysis."""
+
+    container_prefix = "sb"
+    id = Field(-1, "Scheduling block ID", type=np.int64)
+    type = Field(
+        SchedulingBlockType.OBSERVATION,
+        description="Type of scheduling block",
+        type=SchedulingBlockType,
+    )
+    observing_mode = Field(ObservingMode.CUSTOM, "Observing mode", type=ObservingMode)
```

Comment:
I think we should always add an `UNKNOWN` -1 value here, not one of the actual possible values